ci-operator/config/openshift/installer/openshift-installer-master__altinfra-periodics.yaml

```diff
@@ -106,6 +106,15 @@ tests:
         keyC valueC
     workflow: openshift-e2e-aws-sharednetwork
   timeout: 6h0m0s
+- as: e2e-gcp-capi-ovn
+  interval: 8h
+  steps:
+    cluster_profile: gcp
+    env:
+      FEATURE_GATES: '["ClusterAPIInstall=true"]'
+      FEATURE_SET: CustomNoUpgrade
```

Comment:
After discussion late last week with the TRT team, I understood that we want to favor testing the TechPreview feature set--not using individual gates like this. So for the goal of this PR, you should check whether GCP Tech Preview tests are present in the release periodics: 
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml
and 
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__ci-4.16.yaml

It looks like GCP techpreview jobs are in the ci periodics, but not the nightly periodics. So I believe we want to add the gcp periodics to the nightly config. AWS is also missing from the nightly config, so if you can add that at the same time that would be appreciated.